README.md

```diff
@@ -9,6 +9,28 @@
 
 ----
 
+### Quickstart
+
+There are multiple ways to install the `backup-restore-operator`, including via the Cluster Explorer UI in Rancher v2.5+, but this example will focus on using `helm` to install the Helm charts.
+Clone the [repository](https://github.com/rancher/backup-restore-operator), `cd` into it, and execute the following commands:
```

Comment:
I think the normal way it's suggested to install charts is to add the catalog then install vs cloning a repo `helm repo add > helm install`

Reply:
Should we drop this section from the `README` altogether then? The idea is to have a quick development workflow documented, but I also think it's a good idea to not give the impression that `helm install name path/to/chart` is the primary method either.

Reply:
Well, quickstart and development are two different things. This being a helm chart it needs to have proper helm install instructions, that would include adding our repo and then installing. If development docs are needed then add those to a separate section/file for contributing.

Reply:
In that case, I'll drop this Quickstart section then and add a contributing/development file. I think that would work better in the context of this PR.

Reply:
We need the quickstart and it needs to show how to install this chart using our repo.

Reply:
I'll do both then